app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class UsersController < ApiController
+      def rents_index
+        render json: Rent.all, each_serializer: UserRentSerializer
```

Comment:
Here you are displaying all the available Rents, you should only return the `current_user`'s rents only.
Use, `current_user.rents`